src/main/webapp/resources/css/jsfcrud.css

```diff
@@ -0,0 +1,82 @@
+root { 
```

Comment:
I'll defer to @mheppler on this but I assume we might want to merge this CSS into structure.css.

Reply:
We have plenty of other tiny CSS and JS files from various plugins. Nothing new and different to be concerned about. There are formatting questions with how the files are linked in the template, and their placement in the code, that said, these are all details that are waaaaay too granular for this initial prototype phase of code review.